docs/src/modules/spring/pages/value-entity.adoc

```diff
@@ -44,6 +44,18 @@ different existing entities within a Kalix application.
 <3> The `CounterEntity` class should extend `kalix.javasdk.valueentity.ValueEntity`.
 <4> The initial state of each counter is defined as 0.
 
+include::partial$entities-key.adoc[tag=intro]
```

Comment:
It will be easier for us to find back this file if we call it `generate-entity-keys.adoc` instead.

Reply:
indeed, tks